src/pages/ApiCall.jsx

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable react/no-array-index-key */
```

Comment:
-  You have done great so far :+1: :+1: but there are some changes you need to make before moving to the next milestone. :+1: :100: 
- [ ] As required by Microverse, you are not meant to use any linters disable rule on your code base unless otherwise stated. The linters are there to make sure your code follows the best practices, I'll suggest you remove these disable rules and fix the issues. :+1: :smile: :small_airplane: